.github/workflows/update-schedule.yml

```diff
@@ -0,0 +1,46 @@
+name: Update schedule.yaml
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 1'  # Every monday
```

Comment:
Does this open one PR each week? That could be annoying.

Reply:
No it re-uses the existing PR with the current configuration if it got not merged in time.